src/main.ts

```diff
@@ -1,9 +1,17 @@
 import { NestFactory } from '@nestjs/core';
 import { AppModule } from './app.module';
+import { ValidationPipe } from '@nestjs/common';
 
 async function bootstrap() {
   const app = await NestFactory.create(AppModule);
   app.setGlobalPrefix('api');
+  app.useGlobalPipes(
```

Comment:
Adding `ValidationPipe` with configuration to transform incoming payloads and validate their contexts (disallow extra fields etc.)